Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workspace/TextFile.java

```diff
@@ -73,6 +74,14 @@ public void setRefId(String refId) {
         this.refId = refId;
     }
 
+    public String getAccessibility() {
```

Comment:
This way it is very clear to everybody who uses your class what values this field can have.